edmq_backend/src/scripts/generate_song_list.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import json
+import shazam_data
+
+# Youtube API Request
```

Comment:
add overview comment on what this script does

Reply:
Edited overview comment, see above.